### PR TITLE
Improved AddIngredient tests

### PR DESCRIPTION
**Pull request by hiyingnn**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -58,6 +58,11 @@
     public static final String INGREDIENT_UNIT_DESC_TOMATO =
             " " + PREFIX_INGREDIENT_UNIT + INGREDIENT_VALID_UNIT_TOMATO;
 
+    public static final String INGREDIENT_INVALID_NAME_DESC = " " + PREFIX_INGREDIENT_NAME
+            + "1"; // integers not allowed in ingredientName
+    public static final String INGREDIENT_INVALID_UNIT_DESC = " " + PREFIX_INGREDIENT_UNIT
+            + "potato"; // characters not allowed in names
+
```
Comment (lingyanhao) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:88]: Try to include negative values like "-1" in your tests too

File: src/main/java/seedu/address/logic/parser/AddIngredientCommandParser.java [hunk 2]
```diff
@@ -20,15 +20,15 @@
 
     public AddCommand parse(String args) throws ParseException {
```
Comment (lingyanhao) [inline src/main/java/seedu/address/logic/parser/AddIngredientCommandParser.java:23]: Try to use [at] override, better coding practice

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (hiyingnn) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java:88]: Edited a new invalid case for negatives.

File: src/main/java/seedu/address/logic/parser/AddIngredientCommandParser.java (hunk 2 above)
Comment (hiyingnn) [reply src/main/java/seedu/address/logic/parser/AddIngredientCommandParser.java:23]: Added this too! thanks for pointing out

**Pull request merged**